garak/analyze/report_digest.py

```diff
@@ -439,74 +424,92 @@ def build_digest(report_filename: str, config=_config):
     if calibration_used:
         report_digest["meta"]["calibration"] = _get_calibration_info(calibration)
 
-    return report_digest
+        report_digest["results"] = extract_flattened_modules(report_digest, config=config)
+
+        # --- Inject digest into standalone HTML file ---
+        template_path = "/Users/opadovani/Documents/Work/garak/garak/analyze/ui/index.html"
+        output_path = f"/Users/opadovani/Documents/Work/garak/garak/analyze/ui/reports/{init['run_uuid']}.html"
```

Comment:
There should not be any hardcoded paths in the python code.

```suggestion
        template_path = _config.transient.package_dir / "ui" / "index.html"
```

The `build_digest()` method is not the right location for writing the final report.  This method should simply return the json that will be injected.